index.js

```diff
@@ -61,20 +37,21 @@ async function createLabel() {
 
   // TODO: Handle case when label already exists as this currently thows an error
   const response = await linearClient.issueLabelCreate({ name: labelName });
-  core.info(`Created label ${labelName}`);
+  core.info(`Created label ${labelName}.`);
 
   const labelUrl = `https://linear.app/perdoo/team/ENG/label/${encodeURIComponent(
     labelName
   )}`;
 
   core.setOutput("url", labelUrl);
-  core.setOutput("label-created", Boolean(labelName));
+  core.setOutput("label-created", true);
 
   return response._issueLabel.id;
 }
 
 async function updateIssues(issues, newLabelId) {
-  each(issues.nodes, async (issue) => {
+  for (let i = 0; i < issues.nodes.length; i++) {
```

Comment:
Has to be this loop, otherwise async doesn't work as expected. In `each`, `forEach`, etc. the callback gets executed like an async function without `await`.

Reply:
Not quite following? I used an `await` in line 81:
`const labels = await issue.labels();`

Reply:
Yes, but there's no `await` on the callback function.

Reply:
It's a loop, there's no callback? Do you mean `issue.update(payload);`?

`for` loops like this feel a bit dated, there must be a way to use async await with something easier. But it's not a big deal, not worth much more time.

Reply:
No, i mean `each(issues.nodes, async (issue) => {...});`. `async (issue) => {...}` gets called without an `await`. It's described here: https://zellwk.com/blog/async-await-in-loops/